imagersite/imager_profile/forms.py

```diff
@@ -0,0 +1,30 @@
+from .models import ImagerProfile
+from django.forms import ModelForm
+
+
+class ProfileEditForm(ModelForm):
+    """Instantiate user profile edit forms."""
+    class Meta:
+        model = ImagerProfile
+        fields = [
+            'bio',
+            'phone',
+            'location',
+            'website',
+            'fee',
+            'camera',
+            'services',
+            'photostyles']
+
+    def __init__(self, *args, **kwargs):
+        username = kwargs.pop('username')
+        super().__init__(*args, **kwargs)
+        user = ImagerProfile.objects.get(user__username=username)
```

Comment:
The `ImagerProfile` object being edited is already stored in `self.instance`, you do not need to retrieve it like this.